### ISSUE TITLE
Scope is not persisted on Windows

### ISSUE THREAD

**Issue by timfish** [getsentry/sentry-electron#139]
Affected versions: >= 0.12.1

Not sure how I missed this before but in both 0.12.1 and 0.13, `scope.json` is not getting saved in the `sentry` directory within the app `userData` directory.

**Comment by timfish** [comment]
`'@sentry/utils/store'` `.flush()` is failing and the error is silently swallowed [here](https://github.com/getsentry/sentry-javascript/blob/1fa2f42e811281e1fb018502811216dd0bafbdee/packages/utils/src/store.ts#L85).

When running the e2e tests, the ignored error is below and I'm guessing the issue is the leading forward slash:
```
Error: ENOENT: no such file or directory, mkdir '/C:\Users\tim\AppData\Local\Temp\tmp-cjnzaurx00000xslxs26x5zxe\sentry'  
    at fs.mkdirSync (fs.js:872:3)                                                                                                
    at Object.fs.mkdirSync (ELECTRON_ASAR.js:727:16)                                                                             
    at Object.mkdirpSync (C:\Users\tim\Documents\Repositories\sentry-electron\node_modules\@sentry\utils\fs.js:151:18)           
    at Store.flush (C:\Users\tim\Documents\Repositories\sentry-electron\node_modules\@sentry\utils\store.js:68:12)               
    at Immediate._onImmediate (C:\Users\tim\Documents\Repositories\sentry-electron\node_modules\@sentry\utils\store.js:33:15)    
    at runCallback (timers.js:696:18)                                                                                            
    at tryOnImmediate (timers.js:667:5)                                                                                          
    at processImmediate (timers.js:649:5)                                                                                        
    at process.topLevelDomainCallback (domain.js:121:23)                                                                                                          
```

I think it was broken by [this commit](https://github.com/getsentry/sentry-javascript/commit/94de4002f5a722fd5e69ad21a2b865ca7887b31e) (cc. @kamilogorek) because [this](https://github.com/getsentry/sentry-javascript/blob/1fa2f42e811281e1fb018502811216dd0bafbdee/packages/utils/src/path.ts#L137) is not compatible with Windows.

**Comment by timfish** [comment]
#140 Added a failing test

**Comment by kamilogorek** [comment]
Related: https://github.com/getsentry/sentry-javascript/pull/1712

**Comment by timfish** [comment]
This was fixed in [v0.14.0](https://github.com/getsentry/sentry-electron/releases/tag/v0.14.0).
